app/services/search_item_res.rb

```diff
@@ -38,33 +39,45 @@ def combine_highlights
     end
   end
 
+  def format_bucket_value(facets, field, bucket)
+    # dates return in wonktastic ways, so grab key_as_string instead of gibberish number
+    # but otherwise just grab the key if key_as_string unavailable
+    key = bucket.key?("key_as_string") ? bucket["key_as_string"] : bucket["key"]
+    val = bucket["doc_count"]
+    source = key
+    # top_matches is a top_hits aggregation which returns a list of terms
+    # which were used for the facet.
+    #   Example: "Willa Cather" and "WILLA CATHER"
+    # Those terms will both have been normalized as "willa cather" but
+    # we will want to display one of the non-normalized terms instead
+    matches = bucket.dig("top_matches", "hits", "hits")
+    if matches
+      # elasticsearch stores nested source results without the "path"
+      no_nesting = field.split(".").last
+      source = matches.first.dig("_source", no_nesting)
```

Comment:
I think I understand what's happening here and wonder if `no_nesting` may be confusing. It's just taking the latter part of a nested field after the `.`, right? And "path" is the terminology for the parent in the nested mapping? Maybe `nesting_sans_path`, `nesting_sans_parent`, `nesting_child`, or similar will be a clearer name?

Reply:
Yeah, I get `path` from elasticsearch's own terminology, but the no_nesting was basically trying to say like "the field name itself without considering nesting."  I like your idea of `nesting_child` and I can use that